.backportrc.json

```
@@ -0,0 +1,4 @@
+{
+  "prTitle": "[{{targetBranch}}] {{commits.0.sourcePullRequest.title}}",
```

`commits.0.sourcePullRequest.title` works but I added an alias to get the source pull request of the first commit (which is what you normally want): 

```suggestion
  "prTitle": "[{{targetBranch}}] {{sourcePullRequest.title}}",
```

---

Oh, that's nicer. Thanks for your feedback! Updated :slightly_smiling_face: